com.unity.render-pipelines.high-definition-config/Runtime/ShaderConfig.cs

```diff
@@ -4,40 +4,67 @@
 
 namespace UnityEngine.Rendering.HighDefinition
 {
+    /// <summary>
+    /// Probe Volume Evaluation Mode.
+    /// </summary>
     [GenerateHLSL(PackingRules.Exact)]
     public enum ProbeVolumesEvaluationModes
     {
+        /// <summary>Probe volumes are disabled.</summary>
         Disabled = 0,
+        /// <summary>Probe volumes are evaluated in the light loop.</summary>
```

Comment:
Evaluates probe volumes in the light loop.

Reply:
Are there any remarks to add on the benefits of running probes in the light loop or in the material pass?

Reply:
Some advanced stuff is a bit different but as it stands all this is very much experimental so one of the mode will probably go away anyway so I'm not sure we need to put more effort into this right now :)